dotCMS/src/main/java/com/dotmarketing/beans/transform/TreeTransformer.java

```diff
@@ -1,41 +1,35 @@
-package com.dotmarketing.factories;
+package com.dotmarketing.beans.transform;
 
+import com.dotcms.util.transform.DBTransformer;
 import com.dotmarketing.beans.Tree;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 import com.google.common.collect.Lists;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.Lists'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)